scripts/install-cni.sh

```diff
@@ -145,7 +145,9 @@ if [ "$ENABLE_MASQUERADE" == "true" ]; then
   fi
 fi
 
-if [ "${ENABLE_PRIVATE_IPV6_ACCESS:-}" == "true" ] || [ "$ENABLE_IPV6" == "true" ]; then
+STACK_TYPE=$(printf '%s' "$response" | jq '.metadata.labels."cloud.google.com/gke-stack-type"')
+
```

Comment:
Would be good to add a log stating the stack type on the node. This can help debug issues where we aren't seeing the correct stack type or if we are enabling v6 due to the stack type.

Reply:
Added log line for stack type.